src/main/java/com/uid2/shared/encryption/AesCbc.java

```diff
@@ -44,6 +67,15 @@ public static byte[] decrypt(byte[] encryptedBytes, EncryptionKey key) {
         }
     }
 
+    public static byte[] decrypt(byte[] encryptedBytes, KeysetKey key) {
```

Comment:
with these and below, can we make reusable functions? eg pass key.GetKeyBytes() into a common function that is used by new KeysetKeys and old keys